pyERGM/metrics.py

```diff
@@ -727,6 +727,42 @@ def __str__(self):
         return "node_attribute_in"
 
 
+class SumDistancesConnectedNeurons(ExWeightNumEdges):
+    """
+    This function calculates the sum of euclidean distances between all pairs of connected neurons.
+    rows = samples, columns = axes
+    """
+
+    def __init__(self, exogenous_attr: Collection, is_directed: bool):
+        # I am not sure what is the meaning of defining exogenous_attr as collection, I just saw it was done
```

Comment:
You can specify whatever you want, especially if you validate that later and raise an error unless the type is as expected